colabs/Similarity_tests.ipynb

```diff
@@ -0,0 +1,1197 @@
+            "\n",
+            "--2023-01-22 20:11:24--  https://datasets.imdbws.com/title.ratings.tsv.gz\n",
+            "Resolving datasets.imdbws.com (datasets.imdbws.com)... 13.226.210.101, 13.226.210.114, 13.226.210.25, ...\n",
+            "Connecting to datasets.imdbws.com (datasets.imdbws.com)|13.226.210.101|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 6378252 (6.1M) [binary/octet-stream]\n",
+            "Saving to: ‘title.ratings.tsv.gz’\n",
+            "\n",
+            "title.ratings.tsv.g 100%[===================>]   6.08M  --.-KB/s    in 0.1s    \n",
+            "\n",
+            "2023-01-22 20:11:24 (52.9 MB/s) - ‘title.ratings.tsv.gz’ saved [6378252/6378252]\n",
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%time\n",
+        "train_data = pd.read_parquet(TRAIN_DATA, columns=[\"nconst\", \"tconst\", \"category\", \"primaryName\", \"knownForTitles\",\n",
+        "                                                  \"no_nominations_oscars_norm\", \"no_nominations_globes_norm\",\n",
+        "                                                  \"no_nominations_emmy_norm\", \"no_films_norm\",\n",
+        "                                                  \"average_films_rating_norm\", \"genres_code\", \"types_code\"])\n",
+        "validate_data = pd.read_parquet(VALIDATE_DATA, columns=[\"nconst\", \"tconst\", \"category\", \"primaryName\", \"knownForTitles\",\n",
+        "                                                  \"no_nominations_oscars_norm\", \"no_nominations_globes_norm\",\n",
+        "                                                  \"no_nominations_emmy_norm\", \"no_films_norm\",\n",
+        "                                                  \"average_films_rating_norm\", \"genres_code\", \"types_code\"])"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "AMzXW0oIGAqT",
+        "outputId": "5e718c27-a83d-4aa2-cccf-e197974db849"
+      },
+      "execution_count": 8,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "CPU times: user 28.8 s, sys: 6.71 s, total: 35.5 s\n",
+            "Wall time: 46.6 s\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Porównywanie dwóch aktorów"
+      ],
+      "metadata": {
+        "id": "r8q5uNEDV57T"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Wyciągnięcie pierwszego aktora"
+      ],
+      "metadata": {
+        "id": "Oum5fGAmGpzt"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "actor1 = \"Natalie Portman\""
+      ],
+      "metadata": {
+        "id": "4E_wuUsfIfVM"
```

Comment:
Chyba łatwiej będzie szukać aktora po jego indeksie `nconst`, bo do tego jest już funkcja. Wystarczy go wydobyć z linku do profilu aktora. Np dla Chucka Norris mamy link: [https://www.imdb.com/name/nm0001569/](https://www.imdb.com/name/nm0001569/) gdzie na końcu jest jego `nconst`

Reply:
Wtedy nie narażamy się na duplikaty po imieniu, a wystarczy w komentarzu napisać jak się nazywa ten aktor

Reply:
hmm myślałam, że lepiej będzie imieniem i nazwiskiem, bo nie mamy jak wyciągnąć id (w sensie nie znamy go). To chyba rzeczywiście lepsze podejście - zmieniłam